packages/now-cli/src/util/dev/server.ts

```diff
@@ -2261,3 +2272,11 @@ function hasNewRoutingProperties(nowConfig: NowConfig) {
     typeof nowConfig.trailingSlash !== undefined
   );
 }
+
+function buildMatchEquals(a?: BuildMatch, b?: BuildMatch): boolean {
+  if (!a || !b) return false;
+  if (a.src !== b.src) return false;
+  if (a.use !== b.use) return false;
+  if (!deepEqual(a.config || {}, b.config || {})) return false;
+  return true;
```

Comment:
Is there a reasono why you are only comparing 3 properties (src, use, config) instead of all?

```ts
function buildMatchEquals(a?: BuildMatch, b?: BuildMatch)
  return deepEqual(a, b);
}
```

Reply:
I guess there are a lot of properties on `BuildMatch` and we only want to compare the properties on `BuildConfig` 👍